jsonnet/dashboards/k8s/container-overview.libsonnet

```diff
@@ -61,6 +61,42 @@ local getNextIndex(arrays) =
           hide='variable',
         );
 
+      local namespaceTemplate =
+        template.new(
+          name='namespace',
+          label='Namespace',
+          query='label_values(kube_pod_container_info{cluster=~"$cluster"}, namespace)',
+          datasource='$datasource',
+          refresh=$._config.grafanaDashboards.templateRefresh,
+          sort=$._config.grafanaDashboards.templateSort,
+          includeAll=true,
+          multi=true,
+        );
+
+      local podTemplate =
+        template.new(
+          name='pod',
+          label='Pod',
+          datasource='$datasource',
+          query='label_values(kube_pod_container_info{cluster=~"$cluster", namespace=~"$namespace", container=~"$container"}, pod)',
```

Comment:
I would switch `podTemplate` and `containerTemplate` and change queries accordingly. Like in [container-detail](https://github.com/dNationCloud/kubernetes-monitoring/blob/main/jsonnet/dashboards/k8s/container-detail.libsonnet#L188)